build.mjs

```diff
@@ -0,0 +1,79 @@
+import fs from "fs";
+import { marked } from "marked";
+
+class CustomRenderer extends marked.Renderer {
+  paragraph(text) {
+    if (typeof text !== "string") return `<p>${text}</p>`;
+    if (text.includes("<img")) return `<p>${text}</p>`;
+    if (text.includes("open source QOL")) return `<p class="first-p">${text}</p>`;
+    return `<p class="first-p">${text}</p>`;
+  }
+
+  link(href, title, text) {
+    return `<a href="${href}" target="_blank">${text}</a>`;
+  }
+
+  image(href, title, text) {
+    return `<img src="${href}" alt="${text}" width="320" height="180" />`;
```

Comment:
would be nice to be able to expand width and height in the future, so add them as parameters